setup.py

```diff
@@ -16,7 +16,7 @@
 
 setup(
     name=PACKAGE_NAME,
-    version='0.0.1',
+    version='0.0.3',
```

Comment:
Should this be 0.0.2?

Reply:
i published a version 0.0.2 with the previous code change, so with this new change it will now be 0.0.3 😄